MODULE.bazel

```diff
@@ -3,27 +3,37 @@ module(
     version = "0.4.0",
 )
 
+bazel_dep(
+    name = "rules_web_testing_go",
```

Comment:
You are creating a cycle here. Pls have copy here (e.g. web/internal) and symbolic link as we did for go files. (that's why we created symbolic link for those)